lib/generic/modules/metabolic_syndrome_disease.json

```diff
@@ -0,0 +1,651 @@
+      "direct_transition" : "Moderate_Kidney_Damage_Symptom_4"
+    },
+
+    "Moderate_Kidney_Damage_Symptom_4" : {
+      "type" : "Symptom",
+      "symptom" : "Thirst",
+      "range" : { "low" : 20, "high" : 100 },
+      "direct_transition" : "Retinopathy_Progression"
+    },
+
+    "Set_Severe_Kidney_Damage" : {
+      "type" : "SetAttribute",
+      "attribute" : "diabetic_kidney_damage",
+      "value" : 3,
+      "direct_transition" : "Severe_Kidney_Damage_Symptom_1"
+    },
+
+    "Severe_Kidney_Damage_Symptom_1" : {
+      "type" : "Symptom",
+      "symptom" : "Hunger",
+      "range" : { "low" : 40, "high" : 100 },
+      "direct_transition" : "Severe_Kidney_Damage_Symptom_2"
+    },
+
+    "Severe_Kidney_Damage_Symptom_2" : {
+      "type" : "Symptom",
+      "symptom" : "Fatigue",
+      "range" : { "low" : 40, "high" : 100 },
+      "direct_transition" : "Severe_Kidney_Damage_Symptom_3"
+    },
+
+    "Severe_Kidney_Damage_Symptom_3" : {
+      "type" : "Symptom",
+      "symptom" : "Frequent Urination",
+      "range" : { "low" : 40, "high" : 100 },
+      "direct_transition" : "Severe_Kidney_Damage_Symptom_4"
+    },
+
+    "Severe_Kidney_Damage_Symptom_4" : {
+      "type" : "Symptom",
+      "symptom" : "Thirst",
+      "range" : { "low" : 40, "high" : 100 },
+      "direct_transition" : "Retinopathy_Progression"
+    },
+
+    "Set_End_Stage_Kidney_Damage" : {
+      "type" : "SetAttribute",
+      "attribute" : "diabetic_kidney_damage",
+      "value" : 4,
+      "direct_transition" : "End_Stage_Kidney_Damage_Symptom_1"
+    },
+
+
+    "End_Stage_Kidney_Damage_Symptom_1" : {
+      "type" : "Symptom",
+      "symptom" : "Hunger",
+      "range" : { "low" : 50, "high" : 100 },
+      "remarks" : [
+        "Without intervention, 20-40 percent of patients with type 2 diabetes/microalbuminuria, will evolve to macroalbuminuria.",
+        "Shlipak, Michael. 'Clinical Evidence Handbook: Diabetic Nephropathy: Preventing Progression - American Family Physician'. www.aafp.org."
+      ],
+      "direct_transition" : "End_Stage_Kidney_Damage_Symptom_2"
+    },
+
+    "End_Stage_Kidney_Damage_Symptom_2" : {
+      "type" : "Symptom",
+      "symptom" : "Fatigue",
+      "range" : { "low" : 50, "high" : 100 },
+      "direct_transition" : "End_Stage_Kidney_Damage_Symptom_3"
+    },
+
+    "End_Stage_Kidney_Damage_Symptom_3" : {
+      "type" : "Symptom",
+      "symptom" : "Frequent Urination",
+      "range" : { "low" : 50, "high" : 100 },
+      "direct_transition" : "End_Stage_Kidney_Damage_Symptom_4"
+    },
+
+    "End_Stage_Kidney_Damage_Symptom_4" : {
+      "type" : "Symptom",
+      "symptom" : "Thirst",
+      "range" : { "low" : 50, "high" : 100 },
+      "direct_transition" : "Chance_of_Death_From_End_Stage_Kidney_Disease"
+    },
+
+    "Chance_of_Death_From_End_Stage_Kidney_Disease" : {
+      "type" : "Simple",
+      "distributed_transition" : [
+        { "distribution" : 0.0001, "transition" : "Death" },
```

Comment:
Not sure this will work correctly based on the configurable time-step... Perhaps we should just figure out how long people can live with ESRD (4-10 years if they are on dialysis, 1-4 weeks without dialysis) and set a future death date.